src/Umbraco.Core/Configuration/Models/ModelsBuilderSettings.cs

```diff
@@ -18,10 +20,23 @@ public class ModelsBuilderConfig
         /// </remarks>
         public bool Enable { get; set; } = false;
 
+        // See note on ContentSettings.MacroErrors
+        internal string ModelsMode { get; set; } = Configuration.ModelsMode.Nothing.ToString();
```

Comment:
Can these be private like we have for connection string? I'm afraid we will use this value where it is not intended.

Reply:
The json parser won't be able to see private fields most likely.

Reply:
Unfortunately not - as I'm looking to do the validation on these string representations, that validator classes need to be able to read them.  As currently do the builders used in the unit tests..